jetty-client/src/main/java/org/eclipse/jetty/client/HttpResponse.java

```diff
@@ -87,6 +87,11 @@ public HttpFields getHeaders()
         return headers.asImmutable();
     }
 
+    protected HttpFields.Mutable headers()
```

Comment:
There is already the `public HttpResponse headers(Consumer<HttpFields.Mutable> consumer)` API that allows mutating the headers. IMHO one of these two methods should go.